notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<blockquote>but that does not take into account the PQ compression; </blockquote><p>It would help to say "does not take into account the PQ compression, which is lossy."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>